lib/msf/core/constants.rb

```diff
@@ -49,6 +49,49 @@ module Msf
     ExcellentRanking => "excellent"
   }
 
+#
+# Stability traits
+#
+
+# Module should not crash the service
+CRASH_SAFE             = 'crash-safe'
+# Module may crash the service, but the service restarts.
+CRASH_SERVICE_RESTARTS = 'crash-service-restarts'
+# Module may crash the service, and the service remains down.
+CRASH_SERVICE_DOWN     = 'crash-service-down'
+# Module may crash the OS, but the OS restarts.
+CRASH_OS_RESTARTS      = 'crash-os-restarts'
+# Module may crash the OS, and the OS remains down.
+CRASH_OS_DOWN          = 'crash-os-down'
+# Module may cause a resource (such as a file or data in database) to be unavailable for the service.
+SERVICE_RESOURCE_LOSS  = 'service-resource-loss'
+# Modules may cause a resource (such as a file) to be unavailable for the OS.
+OS_RESOURCE_LOSS       = 'os-resource-loss'
+
+#
+# Side-effect traits
+#
+
+# Modules leaves payload or a dropper on the target machine
+ARTIFACTS_ON_DISK      = 'artifacts-on-disk'
+# Module modifies some config file on the target machine
+CONFIG_CHANGES         = 'config-changes'
+# Module leaves signs of a compromise in a log file (Example: SQL injection data found in HTTP log)
+IOC_IN_LOGS            = 'inc-in-logs'
```

Comment:
Was `'inc-in-logs'` meant to be `'ioc-in-logs'` ?

Reply:
oh gosh. I got it wrong.